Documentation/Configuration/GlobalVariables.rst

```diff
@@ -14,6 +14,28 @@ $GLOBALS
    has no effect anymore. Use the
    :ref:`ModuleProvider <backend-modules-api>` instead.
 
+.. confval:: LANG
```

Comment:
We should add a note that it is discouraged to use the global variable and the LanguageServiceFactory should be used.

Reply:
done